tests/custom_strategies/test_strategies.py

```diff
@@ -196,3 +199,75 @@ def test_arbitrary_indices_strategy(a, data):
     # if index does not comply with numpy indexing
     # rules, numpy will raise an error
     a[index]
+
+
+@pytest.mark.parametrize("constant", [st.booleans(), None])
+def test_tensors_handles_constant_strat(constant):
+    constants = []
+    kwargs = dict(dtype=np.int8, shape=(2, 3))
+    if constant is not None:
+        kwargs["constant"] = constant
```

Comment:
What is the purpose of the `None`? If `constant` is not specified in the strategy, it uses `st.booleans` anyway. Wouldn't it make more sense to use `[True, False, None]` in the `parametrize` decorator?

Reply:
This just ensures that `None` has the same behavior as specifying `st.booleans()`. The static constant  cases are tested below.

Reply:
Ahh I see :+1: